index.html

```diff
@@ -130,8 +131,288 @@
+sometimes also called CANONICALIZATION.
+        </dd>
+        <dt><dfn>message digest algorithm</dfn></dt>
+        <dd>
+An algorithm that takes an input message and produces a cryptographic
+output message that is often many orders of magnitude smaller than the
+input message. These algorithms are often 1) very fast, 2)
+non-reversible, 3) cause the output to change significantly when even one
+bit of the input message changes, and 4) make it infeasible to find two
+different inputs for the same output.
+        </dd>
+        <dt><dfn>signature algorithm</dfn></dt>
+        <dd>
+An algorithm that takes an input message and produces an output value where the
+receiver of the message can mathematically verify that the message has not
+been modified in transit and came from someone possessing a particular secret.
+        </dd>
+      </dl>
+
+      <section>
+        <h2>Linked Data Signature Terms</h2>
+        <p>
+  The following terms are used to describe concepts involved in the
+  generation and verification of signatures according to EcdsaSecp256k1Signature2019.
+        </p>
+  
+        <dl>
+          <dt><dfn>EcdsaSecp256k1Signature2019</dfn></dt>
+          <dd>
+            A JSON-LD Document has been signed with EcdsaSecp256k1Signature2019, 
+            when it contains a proof field, and that proof field contains a jws, as 
+            is defined by this spec.
+          </dd>
+
+          <dt><dfn>EcdsaSecp256k1VerificationKey2019</dfn></dt>
+          <dd>
+            A secp256k1 public key. If the key is embedded in a controller with property name publicKeyJwk, the key must be a valid JWK. If the key is embeded as publicKeyHex, it must be a compressed public key, and must be converted to JWK before being used to verify signatures according to JWS.
+          </dd>
+        </dl>
+  
+      </section>
+
+    </section>
+
+    <section>
+      <h2>Key Format</h2>
+
+	<section>
+      <h2>Examples</h2>
+
+      <p>
+JWK encoded public key example:
+      </p>
+
+      <pre class="example highlight">
+{
+  "@context": ["https://w3id.org/security/v1"],
+  "id": "did:example:123456789abcdefghi#kid=JUvpllMEYUZ2joO59UNui_XYDqxVqiFLLAJ8klWuPBw",
+  "type": "EcdsaSecp256k1VerificationKey2019",
+  "controller": "did:example:123456789abcdefghi",
+  "expires": "2017-02-08T16:02:20Z",
+  "publicKeyJwk" : {
+    "crv": "secp256k1",
+    "kid": "JUvpllMEYUZ2joO59UNui_XYDqxVqiFLLAJ8klWuPBw",
+    "kty": "EC",
+    "x": "dWCvM4fTdeM0KmloF57zxtBPXTOythHPMm1HCLrdd3A",
+    "y": "36uMVGM7hnw-N6GnjFcihWE3SkrhMLzzLCdPMXPEXlA",
+  }
+}
+      </pre>
+
+      <p>Note: The <code>publicKeyJwk</code> property contains a key in JWK format.</p>
+
+      <p>
+        Hex encoded public key example:
+              </p>
+        
+              <pre class="example highlight">
+        {
+          "@context": ["https://w3id.org/security/v1"],
+          "id": "did:example:123456789abcdefghi#kid=JUvpllMEYUZ2joO59UNui_XYDqxVqiFLLAJ8klWuPBw",
```

Comment:
Could this be changed to `did:example:123456789abcdefghi#keys-1` or `did:example:123456789abcdefghi#JUvpllMEYUZ2joO59UNui_XYDqxVqiFLLAJ8klWuPBw` or `did:example:123456789abcdefghi#keys-JUvpllMEYUZ2joO59UNui_XYDqxVqiFLLAJ8klWuPBw` to be better aligned with DID specs? Just not something that looks like a key/value pair inside the fragment.

Same for other two occurrences of `#kid=...` in this file.

Reply:
yes, this is not the right place for me to wine about DIDs not using `kid` and rfc7638.

I am personally triggered by the `did:example:123456789abcdefghi#keys-1` syntax most of the times I see it :)